scripts/install_environment.sh

```diff
@@ -171,16 +171,25 @@ function install_runtime_container() {
 
 function install_packages() {
     echo "Installing dnf packages"
-    sudo dnf install -y make python3 python3-pip git jq bash-completion xinetd
+    sudo dnf install -y make git jq bash-completion xinetd
     sudo systemctl enable --now xinetd
 
+    if dnf list --available python3.11; then
+        echo "Installing Python 3.11"
+        sudo dnf install -y python3.11
+        python3 -m ensurepip --upgrade
+    else
+        echo "Python 3.11 is unavailable. Fallback to any version of Python 3 available"
+        sudo dnf install -y python3 python3-pip
```

Comment:
Why do we need that? the latest version should be available, if not I think we should fail

Reply:
Too many options in beaker to choose an OS that doesn't specifically has the ``python-3.11`` package, so I tried to be backwards-compatible